### DISCUSSION TITLE
An error occurred

### DISCUSSION THREAD

**Discussion by brineu** [community/community#124601] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

How to solve it?
![Screenshot 2024-05-19 134938](https://github.com/community/community/assets/110652156/d65804c7-77a3-49c0-83a6-dbad189a3b96)

**Comment by athrv** [comment]
Same error, typically for .ipynb files

**Reply by MangalAdhav** [reply]
Same here.. Since afternoon

**Reply by MangalAdhav** [reply]
How to tackle this?? I am fed up.. I have uninstall all apps and installed them again.. Still same error showing.. Down it s saying using nbformat v5. 10.4 and nbconvert v7. 16.1

**Reply by SaiPrasanna979** [reply]
use this 
https://nbviewer.org/

**Comment by CopperEagle** [comment]
Hey @brineu 

Yes, the platform is likely experiencing an issue with nbconvert. Everybody is affected by this, see #124617. Probably gotta wait until tomorrow for a fix.

**Comment by Adykey79** [comment]
I am still facing this problem

**Comment by BellatrixSalungweni** [comment]
I thought I was the only one facing this problem
